osgar/tools/lidarview.py

```diff
@@ -189,6 +189,54 @@ def next(self):
                 self.history[self.index - self.history_size] = None
         return self.history[self.index]
 
+class Framer:
```

Comment:
Can you explain why name "Framer"? Maybe add doc??

Reply:
I've added a comment - it creates frames.